src/main/java/seedu/address/model/Model.java

```diff
@@ -128,4 +128,9 @@
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredGroupList(Predicate<Group> predicate);
+
+    /**
+     * Assigns a person to a group.
```

Comment:
javadoc can be something like 

Assigns a person to a group.
{@code Group} must already exist in the address book.

To be similar to other methods in this class.